Jenkinsfile

```diff
@@ -0,0 +1,129 @@
+@Library('pipeline-library')
+import com.genesys.jenkins.Service
+
+def notifications = null
+
+def isReleaseBranch() {
+    return env.SHORT_BRANCH.equals('main');
+}
+
+pipeline {
+  agent { label 'dev_mesos_v2' }
+  options {
+    quietPeriod(480)
+    disableConcurrentBuilds()
+  }
+
+  environment {
+    NPM_UTIL_PATH = "npm-utils"
+    REPO_DIR = "repo"
+    SHORT_BRANCH = env.GIT_BRANCH.replaceFirst(/^origin\//, '');
+    NPM_TOKEN = credentials('2844c47b-19b8-4c5f-b901-190de49c0883')
+  }
+
+  tools {
+    nodejs 'NodeJS 12.13.0'
+  }
+
+  stages {
+    stage('Setup mailing list parameter') {
+      steps {
+        script {
+          properties([
+            parameters([
+              string(
+                defaultValue: '',
+                name: 'EMAIL_LIST',
+                trim: true
+              )
+            ])
+          ])
+        }
+      }
+    }
+    stage('Import notifications lib') {
+      steps {
+        script {
+          // clone pipelines repo
+          dir('pipelines') {
+            git branch: 'master',
```

Comment:
I might be wrong, but I think we do not have gir branch `master` in this repo.

Reply:
@AfanasievAnton this isn't pulling code from our repo but from the `pipeline-library` repo. Similar below with `npm-utils`.